freedata_server/wavelog_api_logger.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import re
+
+
+def send_wavelog_qso_data(config, wavelog_data):
+    """
+    Sends wavelog QSO data to the specified server via API call.
+
+    Parameters:
+    server_host:port (str)
+    server_api_key (str)
+    wavelog_data (str): wavelog-formatted ADIF QSO data.
+    """
+    wavelog_host = config['MESSAGES'].get('wavelog_host', 'http://localhost:8086')
+    wavelog_api_key = config['MESSAGES'].get('wavelog_api_key', '')
+
+    url = wavelog_host + "/index.php/api/qso"
+    headers = {
+        "Content-Type": "application/json",
+        "Accept": "application/json"
+    }
+
+    # If there is no gridsquare fix this so that it follows the ADIF standard.
+    wavelog_data = re.sub(r"<GRIDSQUARE:4>----", "<GRIDSQUARE:0>", wavelog_data)
```

Comment:
I think we should check here for more cases, like empty grid, ---- grid, not existing key.